engine/src/main/java/com/datatorrent/stram/StreamingAppMasterService.java

```diff
@@ -608,13 +609,16 @@ protected void serviceStart() throws Exception
 
     try {
       Configuration config = getConfig();
+      YarnConfiguration conf = new YarnConfiguration();
```

Comment:
As `Configuration` is a super class for `YarnConfiguration`, `Configuration` is not needed by WebApp server, it may use `YarnConfiguration`. Please simplify code to avoid creating multiple Configuration/YarnConfiguration objects and reading .conf files multiple times.